e2e/tests/playwright.config.ts

```diff
@@ -1,14 +1,15 @@
 import type { PlaywrightTestConfig } from '@playwright/test'
 import { devices } from '@playwright/test'
-import { CloudfrontUrls, getCloudfrontUrls } from './src/cloudfront'
 import { getProjectName } from './src/project'
+import { CloudfrontUrls, getCloudfrontUrls } from './src/utils/cloudfront'
 
 /**
  * Read environment variables from file.
  * https://github.com/motdotla/dotenv
  */
 // require('dotenv').config();
 
+//const cloudfrontUrls = Object.entries(getCloudfrontUrls())
```

Comment:
nit: seems like it is safe to remove, as it is the same code as below

Reply:
Removed, thanks!